docs/reference/mapping/runtime.asciidoc

```diff
@@ -752,6 +761,79 @@ The API returns the following result. Without building your data structure in
 advance, you can search and explore your data in meaningful ways to experiment
 and determine which fields to index.
 
+Also, remember that `if` statement in the script?
+
+[source,txt]
+----
+if (clientip != null) emit(clientip);
+----
+
+If the script didn't include this condition, the result wouldn't include any
+documents. By including this condition, the script still returns documents if
```

Comment:
I think it'd be a little more correct to say "the query would fail on any shard that doesn't match the pattern." Adding the `if` skips data that isn't matched.